src/Microsoft.Data.Sqlite.Core/SqliteCommand.cs

```diff
@@ -322,6 +322,11 @@ public new virtual SqliteDataReader ExecuteReader(CommandBehavior behavior)
                     ? PrepareAndEnumerateStatements()
                     : _preparedStatements)
                 {
+                    if (!unprepared)
+                    {
+                        raw.sqlite3_reset(stmt);
```

Comment:
Oooooooh, I see now. For some reason I assumed this call was `sqlite3_reset_bindings()`. I should actually look at the code when reviewing. 😉

In that case, and we do this when we decide not to pass it to the reader?